lib/facebook.ex

```diff
@@ -493,6 +522,73 @@ defmodule Facebook do
       |> summary_count_all
   end
 
+  @doc """
+  Gets payment info about a single payment.
+
+  ## Examples
+      iex> Facebook.payment("769860109692136", "<App Access Token>", "id,request_id,actions")
+      {:ok, %{"request_id" => "abc2387238", "id" => "116397053038597", "actions" => [ %{ "type" => "charge", ... } ] } }
+
+  See:
+    * https://developers.facebook.com/docs/graph-api/reference/payment
+  """
+  @spec payment(object_id, access_token, fields) :: resp
+  def payment(payment_id, access_token, fields \\ "") do
+    params = [fields: fields]
+               |> add_access_token(access_token)
+
+    ~s(/#{payment_id})
+      |> GraphAPI.get([], params: params)
+      |> ResponseFormatter.format_response
+  end
+
+  @doc """
+  Settle a payment dispute.
+
+  ## Examples
+      iex> Facebook.dispute("769860109692136", "<App Access Token>", :DENIED_REFUND)
+      {:ok, %{"success" => true}}
+
+  See:
+    * https://developers.facebook.com/docs/graph-api/reference/payment/dispute
+  """
+  @spec dispute(object_id, access_token, dispute_reason) :: resp
+  def dispute(payment_id, access_token, reason) do
```

Comment:
I'd prefer that having the name `payment_dispute` to make it clear when reading code.

Reply:
Good point